app/models/volunteer.rb

```diff
@@ -0,0 +1,4 @@
+class Volunteer < ActiveRecord::Base
+  belongs_to :user, class_name: User, primary_key: "id", foreign_key: "user_id"
```

Comment:
@SeanLF looks like these are the defaults. You might be able to get away with just

    belongs_to :user